### PR TITLE
feat: implement BloomFilters endpoint

### PR DESCRIPTION
**Pull request by nicosantangelo**
TBD what addresses to return on /owners-bloom-filter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,153 @@
...
+
 #### Example
 
+```bash
+# Try to find an asset for an invalid registry
+/v1/registry/invalidthings/address/0xc04528c14c8ffd84c7c1fb6719b4a89853035cdd/assets/0xb794f5ea0ba39494ce839613fffba74279579268:2
+{
+  "id": "0xb794f5ea0ba39494ce839613fffba74279579268:2",
+  "amount": 0,
+  "urn": {
```
Comment (nachomazzara) [inline README.md:82]: I thought that the response was `"urn": null` but I am ok if this is better

File: README.md [hunk 2]
```diff
@@ -1,32 +1,153 @@
...
+
+This example uses the [BloomFilter](https://www.npmjs.com/package/bloom-filters#export-and-import) library to get all the available owners and return a JSON response that can later be used like this:
+
+```ts
+const url = '/v1/registry/cryptoregistry/owners-bloom-filter'
+
+const response = await fetch(url).then((res) => res.json())
+const filter = BloomFilter.fromJSON(response)
+
+filter2.has('0xf8af76decf64f4164f0c8c9d38f3fb4781e61c0f') // true
```
Comment (nachomazzara) [inline README.md:130]: maybe? 
```suggestion
filter.has('0xf8af76decf64f4164f0c8c9d38f3fb4781e61c0f') // true
```

File: README.md [hunk 3]
```diff
@@ -1,32 +1,153 @@
...
+filter2.has('0xd357f1ff39dd407b5F383806E025eFeF5ea00F9E') // false
```
Comment (nachomazzara) [inline README.md:131]: same here:
```suggestion
filter.has('0xd357f1ff39dd407b5F383806E025eFeF5ea00F9E') // false
```

File: README.md [hunk 4]
```diff
@@ -1,32 +1,153 @@
...
+
+```bash
+# Try to get the bloom filter for an invalid
+/v1/registry/invalidstuff/owners-bloom-filter
+{}
+
+# Try to get the bloom filter for a valid registry
+/v1/registry/cryptoregistry/owners-bloom-filter
+{
+  "type": "BloomFilter",
```
Comment (nachomazzara) [inline README.md:144]: Do we need all of these properties? CC @menduz

File: README.md (hunk 1 above)
Comment (nicosantangelo) [reply README.md:82]: If I'm being honest I couldn't find an example in neither the [ADR](https://github.com/decentraland/adr/blob/main/docs/ADR-42-third-party-assets-integration.md#third-party-resolver) nor the [docs](https://docs.decentraland.org/decentraland/linked-wearables/#building-the-api) so I went with the types that were on the project already 😊

File: README.md (hunk 2 above)
Comment (nicosantangelo) [reply README.md:130]: good 👁️

File: README.md (hunk 1 above)
Comment (nachomazzara) [reply README.md:82]: Do you mind adding it to the ADR & Docs once this is merged?

**Review by nachomazzara — APPROVED** [review]

**Pull request merged**